Readme.md

```diff
@@ -3,23 +3,63 @@
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E918)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
 
-* [Default.aspx](./CS/WebSite/Default.aspx) (VB: [Default.aspx](./VB/WebSite/Default.aspx))
-* [Default.aspx.cs](./CS/WebSite/Default.aspx.cs) (VB: [Default.aspx.vb](./VB/WebSite/Default.aspx.vb))
-<!-- default file list end -->
-# How to display progress information about server-side callback processing
+# ASP.NET Web Forms - How to display progress information about server-side callback processing
 <!-- run online -->
 **[[Run Online]](https://codecentral.devexpress.com/e918/)**
 <!-- run online end -->
 
+This example illustrates how you can inform your users about a server-side process when an operation initiated via a callback request takes much time.
+
+![](image.png)
+
+In this example, a callback panel sends a request to the server and performs a long server process. 
+
+Another callback panel sends a callback to the server every timer tick. On the server, a callback event handler gets the current operation progress and return this value to the client.
+
+## How to perform parallel callbacks
+
+This approach does not work if the Session state is used. The Session state blocks parallel execution and forces parallel requests to be executed one after another because the access to ASP.NET Session state is exclusive per session. 
+
+To avoid this issue, you can [disable the Session](https://learn.microsoft.com/en-US/troubleshoot/developer/webapps/aspnet/development/disable-asp-session-state) state in the following ways:
+
+- At page level, set the `EnableSessionState` property to `False`:
+
+    ```aspx
+    <%@ Page Language="C#" AutoEventWireup="true" Inherits="_Default" EnableSessionState="False" CodeFile="Default.aspx.cs" %>
+    ```
+
+    You can also set the page `EnableSessionState` property to `ReadOnly`. In this case, the Session state is enabled but not writable.
+
+- At project level, set the `sessionState` mode key to `off` in the **Web.config** file:
+
+    ```aspx
+    <system.web>  
+        <sessionState mode="Off"></sessionState>  
+    </system.web>
+    ```
+
+Note that when the Session state is disabled, it is no longer possible to use Session variables. The states below can be used as alternatives:
+- Application state that stores variables that can be accessed by all users of an ASP.NET application.
+- Profile properties that persists user values in a data store without expiring them.
+- ASP.NET caching that stores values in memory that is available to all ASP.NET applications.
+- Cookies.
+- Query string and fields on an HTML form that are available from an HTTP request.
```

Comment:
one string but several fields? is that correct?

Reply:
```suggestion
- Query string that is available from an HTTP request.
```